package.json

```diff
@@ -5,6 +5,8 @@
   "private": true,
   "scripts": {
     "build": "nx run-many --all -t build",
+    "format:check": "prettier -c .",
+    "format:write": "prettier -w .",
```

Comment:
I don't understand why these were added as this looks like it's overwriting the builtin `nx` setup for formatting. If it's not working the way you want I would expect config changes rather than this.

Reply:
@irahopkinson I agree, but have a couple hang-ups. First, I have some mismatches between the nx setup, the prettier setup, and my vscode setup that I'm still wrestling to reconcile. Second, I'm befuddled by the failure the first commit of this pr hit (https://github.com/sillsdev/lang-font-picker/actions/runs/8711041405/job/23894743625#step:6:8) in the pr check that I don't encounter locally.

Reply:
Gotcha, those 2 issues are likely related or even the same issue.